Sources/ComposableArchitecture/Store.swift

```diff
@@ -105,14 +222,16 @@ public final class Store<State, Action> {
   ///     `LocalState`.
   ///   - fromLocalAction: A function that transforms `LocalAction` into `Action`.
   /// - Returns: A publisher of stores with its domain (state and action) transformed.
-  public func scope<LocalState, LocalAction>(
-    state toLocalState: @escaping (Observable<State>) -> Observable<LocalState>,
+  public func scope<O: ObservableType, LocalState, LocalAction>(
```

Comment:
Note that in PF’s upstream code this and the other slightly-tweaked overrides of `scope` are now called `publisherScope`.